### PR TITLE
chore: add issue templates

### LINKED ISSUE #89: Add issue templates for bug reports and feature requests

### PR DESCRIPTION
**Pull request by mlegner**
Adapted from the [walrus-sites](https://github.com/MystenLabs/walrus-sites) repository.

Closes #89

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  - type: checkboxes
+    id: version
+    attributes:
+      label: Version
+      description: >
+        Have you made sure that you are using the latest versions of the `walrus` binary
+        and the configuration?
+      options:
+        - label: My `walrus` binary and configuration are up to date.
+          required: true
```
Comment (Tzal3x) [inline .github/ISSUE_TEMPLATE/bug.yaml:66]: Are these sections relevant to walrus docs? If it's a bug regarding the sites, an issue should be opened there, no?

**Review by Tzal3x — COMMENTED** [review]
Nice initiative! I just left a question about bug reports.

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 1 above)
Comment (mlegner) [reply .github/ISSUE_TEMPLATE/bug.yaml:66]: This is about bugs related to Walrus. As the Walrus repo itself is not public, we collect those in this repo.

**Review by giac-mysten — APPROVED** [review]
Very cool, thanks @mlegner . In general, i find the `[Item name]` title to be redundant with the labels, but otherwise looks good!

**Comment by mlegner** [comment]
> Very cool, thanks @mlegner . In general, i find the `[Item name]` title to be redundant with the labels, but otherwise looks good!

Yes, I'm not too sure about them myself. I will remove them. 👍
